test/unit/datatypes/util.py

```diff
@@ -0,0 +1,51 @@
+import os
+import shutil
+import tempfile
+from contextlib import contextmanager
+
+from galaxy.datatypes.sniff import get_test_fname
+from galaxy.util.bunch import Bunch
+from galaxy.util.hash_util import md5_hash_file
+
+
+@contextmanager
+def get_dataset(file, index_attr='bam_index', dataset_id=1, has_data=True):
```

Comment:
`file` is a Python reserved word, can you use a different variable name? Also in `get_input_files()`.